index.html

```diff
@@ -13,7 +13,7 @@
 <body>
     <div class="container">
         <div class="card1">
```

Comment:
- [ ] Please you should use a section tag as a wrapper for the work section since it represents sectioning content it would be semantically better.